app/src/main/java/edu/metrostate/cardealer/activity/DealerInfoActivity.java

```diff
@@ -0,0 +1,111 @@
+package edu.metrostate.cardealer.activity;
+
+
+import android.content.Intent;
+import android.os.Bundle;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.TextView;
+
+
+import java.io.Serializable;
+
+import edu.metrostate.cardealer.CarDealerApplication;
+import edu.metrostate.cardealer.R;
+import edu.metrostate.cardealer.functionality.VehicleJSONParser;
+import edu.metrostate.cardealer.inventory.Dealership;
+import edu.metrostate.cardealer.storage.StateManager;
+
+public class DealerInfoActivity extends AppCompatActivity {
+
+    @Override
+    protected void onPause() {
+        super.onPause();
+
+        StateManager.save();
+    }
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_dealer_info);
+        Bundle selectedDealer = getIntent().getExtras();
+        Dealership workingDealer = StateManager.dealerGroup.getDealerByID(selectedDealer.getString("workingDealerID"));
+
+
+        TextView dealerID = findViewById(R.id.dealerID);
+        dealerID.setText(getString(R.string.Dealer_ID, workingDealer.getDealerID()));
+        TextView dealerName = findViewById(R.id.dealerName);
+        dealerName.setText(getString(R.string.Dealer_Name, workingDealer.getName()));
+
+        TextView dealerRenting = findViewById(R.id.dealerRenting);
+        dealerRenting.setText(getString(R.string.Dealer_Renting, workingDealer.isRenting()));
+
+        TextView dealerAcquisition = findViewById(R.id.dealerAcquisition);
+        dealerAcquisition.setText(getString(R.string.Dealer_VehicleAcquisition, workingDealer.isVehicleAcquisition()));
+
+        TextView dealerExport = findViewById(R.id.dealerExport);
+        dealerExport.setText(getString(R.string.Dealer_Export));
+
+        //transferring inventory
+        EditText transferInventory = findViewById(R.id.recipient);
+        Button transfer = findViewById(R.id.transferTo);
+        transfer.setOnClickListener(v -> StateManager.dealerGroup.transferInventory(workingDealer.getDealerID(), transferInventory.getText().toString()));
+
+
+        //exporting inventory information to JSON
+        Button exportJSON = findViewById(R.id.JSON);
+        exportJSON.setOnClickListener(v -> {
+            VehicleJSONParser.write(workingDealer);
+            //this is sending the application back to the DealerListActivity for some reason
```

Comment:
its crashing due to an incorrect directory! But I'm fixing this in my branch. 
It might be good to include a dialog here just informing the user that the export occurred